chain-signatures/contract-sol/Anchor.toml

```diff
@@ -0,0 +1,21 @@
+[features]
+seeds = false
+
+[programs.localnet]
+chain_signatures_project = "4kZoBXmUBLveRS3sboGF557tYsR7SzLDsWmP4sz7VQEs"
```

Comment:
I saw that it is possible to give Solana contracts human readable names. Not a blocker, since we will test it on testnet for now.